src/cf-layout.less

```diff
@@ -1606,6 +1606,130 @@
 }
 
 
+
+/* topdoc
+  name: Hero
+  family: cf-layout
+  patterns:
+    - name: .hero
+      markup: |
+        <section class="content_hero hero">
+            <div class="hero_wrapper">
+                <div class="hero_text">
+                    <h1 class="hero_heading">Hero title</h1>
+                    <p class="hero_subhead">
+                        Hero text goes here
+                    </p>
+                    <p class="hero_cta">
+                        <a>Call to action</a>
+                    </p>
+                </div>
+                <div class="hero_image" style="background-image: url('http://placekitten.com/g/400/195')"></div>
+            </div>
+        </section>
+      notes:
+        - "Hero consists of a headline, a small amount of additional text, an optional call to
+           action, and an illustration. Its background color is flush with
+           the sides of the screen, and the content is centered with wrapper classes."
+        - "The illustration can be customized by setting the background-image property
+           on the .hero_wrapper_image element."
+        - "On small screens (or where media queries are not supported), the text spans the
+           full width of the .hero_wrapper and the illustration is displayed underneath."
+        - "For larger screen sizes, media queries are used to position the illustration
+           to the right of the text."
+        - "On desktop, the hero should not exceed 285px in height. The image should be 195px in
+           height to conform to this standard."
+      codenotes:
+        - .hero
+  tags:
+    - cf-layout
+*/
+
+@hero-desktop-height: 285px;
+@hero-image-height: 195px;
+@desktop-min: 801px;
+
+.hero {
+      background-color: @block__bg;
+
+      &_wrapper {
+            box-sizing: border-box;
+            padding: unit(@grid_gutter-width / @base-font-size-px, em)
+                     unit(15px / @base-font-size-px, em);
+            width: 100%;
+            
+            .respond-to-min(@tablet-min, {
+                display: table;
+            });
+
+            .respond-to-min(@desktop-min, {
+                padding: unit(45px / @base-font-size-px, em);
+                min-height: unit(@hero-desktop-height / @base-font-size-px, em);
+            });
+
+
```

Comment:
If we don't need left/right padding above small screens, it'd be better to add a max-width mq instead of resetting to a default.